packages/web-fragments/src/elements/reframed/script-execution.ts

```diff
@@ -61,15 +61,15 @@ export function executeScriptsInPiercedFragment(shadowRoot: ShadowRoot, iframe:
 		const iframeDocument = iframe.contentDocument;
 
 		assert(iframeDocument !== null, 'iframe.contentDocument is not defined');
-
-		getInternalReference(iframeDocument, 'body').appendChild(iframeDocument.importNode(script, true));
+		executeInertScript(script, iframeDocument);
```

Comment:
this is good!

And better than what I did in my pending PR recently where I came across another issue in this function: https://github.com/web-fragments/web-fragments/pull/181/files#diff-21354272ac98f61377af9610d215c3621207772c1ad9d0855f648fa621579facR64-R66